static/src/javascripts/bootstraps/enhanced/article.js

```diff
@@ -31,6 +32,7 @@ const modules = {
         const mainColumn = qwery('.js-content-main-column');
         // only render when we have >1000px or more (enough space for ad + most popular)
         if (
+            !config.hasTone('Match reports') &&
```

Comment:
Do you reckon we should update the comment to mention why 'Match reports' is now being checked? Or we could add a condition like: `mainColumn[0].offsetHeight > config.hasTone('Match reports') ? 1700 : 1150` just to catch cases where there will definitely be space for all three components. Then again, perhaps this will get into the fine-tuning weeds that we wish to avoid... 🤔